src/devtools/devtools.tsx

```diff
@@ -467,6 +483,8 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
         {...panelProps}
       >
         <style
+          scoped
```

Comment:
what does this attribute do please? according to the docs, it's a non-standard, deprecated thing?

https://developer.mozilla.org/en-US/docs/Web/API/HTMLStyleElement/scoped

Reply:
I did not realize that it's deprecated now.
The purpose is to make this style with no side effects on the whole document.

Reply:
I still think we should remove it please.

Reply:
yeah, removed